react/Default.ts

```diff
@@ -0,0 +1,14 @@
+import { FC } from 'react'
+
+import { useConditionContext } from './ConditionContext'
+
+const Default: FC = ({ children }) => {
+  const { matched } = useConditionContext()
+  if (matched !== false) {
+    return null
+  }
+
+  return children as any
```

Comment:
I know this might be an ingenuous question, but why do it this way instead of

```tsx
return (
  <Fragment>
    {children}
  <Fragment />
)
```

Reply:
To prevent an unneeded fragment 👀